training.md

```diff
@@ -89,34 +167,44 @@ Let's make some dbt models! The following steps will show how `dim_customers`, a
-3. From the menu on the left, click on Sources, then shopify_raw, then customers to see the documentation for the raw customers table.
-4. Click on some other tables to view documentation for those
-5. Click on the teal circle icon on the bottom right to view the DAG dbt generates for us. Click on the full screen icon. At the bottom, delete the entry for the `--select` field and press enter to view the full DAG for the entire database you built. Here you can see the path of each table and understand how each table is related to eachother.
+2. Run `dbt docs serve`
+   * Those two commands will take you to a website where you can easily visualize the documentation of each table.
+3. From the menu on the left, click on "Sources", then "shopify_raw", then "customers" to see the documentation for the raw customers table.
+4. Click on some other tables to view documentation for those.
+5. Click on the teal circle icon on the bottom right to view the DAG that dbt generates for us. Click on the full screen icon. At the bottom, delete the entry for the `--select` field and press enter to view the full DAG for the entire database you built. Here you can see the path of each table and understand how each table is related to eachother.
+
+#### 3. Run Tests on Columns
 
-#### Run Tests (dbt test)
 After creating tables, we can run tests on them to see if each column is behaving as expected.
+
 1. Open `models\marts\core\schema.yml`
 2. Here you can see that there is only one test for `dim_customers` and many tests for `fct_orders`. You can look at a list of all tests that can be applied [here](https://docs.getdbt.com/reference/resource-properties/tests/).
+   * Some tests to note from the file are
+      1. `unique`: this test passes when there are no duplicates in the column
+      2. `not_null`: this test passes when there are no null values in the column
+      3. `relationships`: this test passes when all records in a child table have a corresponding record in a parent table.
 3. Run `dbt test` to see which tests pass or fail. A fail will tell you where in your data you need to start debugging.
 
+### Additional Resources
 
-## 2. Learning dbt from better teachers than Anders
-
-Now you're cut loose to learn from the best. [Here's a video](https://www.loom.com/share/c0e55d1734c849c184b1dd3b7df83f02) of me giving a partial intro to these resources
+<!-- Now you're cut loose to learn from the best. [Here's a video](https://www.loom.com/share/c0e55d1734c849c184b1dd3b7df83f02) of me giving a partial intro to these resources
 
-here's where our special [`dim_customers.sql`](dim_customers.sql) lives (theirs doesn't work on TSQL)
+here's where our special [`dim_customers.sql`](dim_customers.sql) lives (theirs doesn't work on TSQL) -->
 
 - [Kyle's amazing course](https://courses.getdbt.com)
 - [Claire's classic text + video walkthrough](https://docs.getdbt.com/tutorial/setting-up)
```

Comment:
is there a way we could more cleanly hand off people into Claire's (or Kyle's) tutorial? also, delete the commented out section probably